tests/mock_modules/testutils/fixtures_data.py

```diff
@@ -258,6 +258,19 @@ def get_summary(secure_arm=False, partition_ids=None,
                         'zone_type': 102,
                         'partition_id': 1,
                     },
+{
+                        'id': '002-0080',
```

Comment:
Let's maybe use `002-0021` since it'll go well with the heat sensor, you can also move the whole block nearby and change the `zone_id` to `221`!
I tried to keep things "sort of together" by category, when possible (I might have to reorganize that sometimes!)